### PR TITLE
fix: use blacklist for parallax

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
Use blacklist for determining device to show parallax animation to.

This solves the issue as noted here https://discord.com/channels/1103692771585433630/1103692773363810317/1210598484076535838 which was earlier raised by John

status: ready

### CODE REVIEW

**Review by cammellos — APPROVED** [review]

File: src/status_im/common/parallax/blacklist.cljs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+(ns status-im.common.parallax.blacklist
+  (:require
+    [native-module.core :as native-module]))
+
+(def ^:const device-id (:device-id (native-module/get-device-model-info)))
```
Comment (ilmotta) [inline src/status_im/common/parallax/blacklist.cljs]: Nitpick: we don't need to use `:const` metadata as much as we think we need. The only difference is that the CLJS compiler will inline `device-id` whenever it sees it. But `device-id` is currently being used only by the static value `blacklisted?`. Same for `minimum-device-code`. So it ends up being noise for no tangible value.

File: src/status_im/common/parallax/blacklist.cljs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+(defn get-model-code
+  [model]
+  (let [match (re-find #"iPhone(\d+)" model)]
```
Comment (ilmotta) [inline src/status_im/common/parallax/blacklist.cljs]: You can destructure to simplify:

```clojure
(let [[_ version] (re-find #"iPhone(\d+)" "iPhone11")]
  version) ; => "11"
```

File: src/status_im/common/parallax/blacklist.cljs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    (if match
+      (js/parseInt (second match))
+      0)))
+
+(def ^:const minimum-device-code 11)
+
+(def blacklisted?
```
Comment (ilmotta) [inline src/status_im/common/parallax/blacklist.cljs:15]: We can have all other vars except this one made private.

**Review by ilmotta — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
Hi @BalogunofAfrica !
Thanks for your PR, tested and ready to be merged!

**Pull request merged**